TodoApp/tests/TodoApp.Services.Tests/Wrappers/DateTimeProviderTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Threading;
+using NUnit.Framework;
+using TodoApp.Contracts.Wrappers;
+using TodoApp.Services.Wrappers;
+
+namespace TodoApp.Services.Tests.Wrappers
+{
+    [TestFixture]
+    internal class DateTimeProviderTests
+    {
+        private IDateTimeProvider _dateTimeProvider;
+
+        [SetUp]
+        public void Init()
+        {
+            _dateTimeProvider = new DateTimeProvider();
+        }
+
+        [Test]
+        public void ReturnsDifferentDateTimes_OnMultipleCalls()
+        {
+            var time1 = _dateTimeProvider.GetCurrentDateTime();
+            Thread.Sleep(1000);
+            var time2 = _dateTimeProvider.GetCurrentDateTime();
+
+            Assert.That(time1, Is.Not.EqualTo(time2));
+        }
+
+        [Test]
+        public void ReturnsApproximatelyCurrentDateTime()
+        {
+            var now = DateTime.Now;
```

Comment:
Here, you rely on an implementation you test. Remove this test completely since you admit only your very one implementation.